content/en/docs/tasks/inject-data-application/define-environment-variable-container.md

```diff
@@ -93,14 +93,14 @@ the list. Similarly, avoid circular references.
 
 ## Using environment variables inside of your config
 
-Environment variables that you define in a Pod's configuration can be used
-elsewhere in the configuration, for example in commands and arguments that
-you set for the Pod's containers.
+Environment variables that you define in a Pod's configuration under 
+`.spec.containers[*].env[*]` can be used elsewhere in the configuration, for 
+example in commands and arguments that you set for the Pod's containers.
 In the example configuration below, the `GREETING`, `HONORIFIC`, and
 `NAME` environment variables are set to `Warm greetings to`, `The Most
-Honorable`, and `Kubernetes`, respectively. Those environment variables
-are then used in the CLI arguments passed to the `env-print-demo`
-container.
+Honorable`, and `Kubernetes`, respectively. And the environment variable 
+`MESSAGE` combines the set of all these  environment variables and then used 
+as a CLI argument passed to the `env-print-demo` container.
```

Comment:
```suggestion
Honorable`, and `Kubernetes`, respectively. The environment variable 
`MESSAGE` combines the set of all these environment variables and then uses it 
as a CLI argument passed to the `env-print-demo` container.
```

Just a suggestion for readability.

Reply:
Hi @stormqueen1990,
Thank you for pointing it out; I've updated the format. PTAL!